setup.py

```diff
@@ -26,6 +26,7 @@
         "eth-hash[pycryptodome]",
         "requests>=2.16.0,<3.0.0",
         "websockets>=4.0.1",
+        "pypiwin32>=223; platform_system == 'Windows'",
```

Comment:
A brief search shows that we may want to remove this whitespace to conform to existing conventions

- pytest: https://github.com/pytest-dev/pytest/blob/master/setup.py#L71-L72
- pipenv: https://github.com/pypa/pipenv/blob/master/setup.py#L30-L32